test/application.js

```diff
@@ -0,0 +1,65 @@
+const { Application } = require('spectron');
```

Comment:
Can we have an option where we can pass the built binary. E.g. if we want to test after we have built the binary for each platform and we want to ensure that all tests pass before publishing it on GitHub.

Reply:
I added an issue for this change #247